src/selectRace/races.ts

```diff
@@ -1,7 +1,7 @@
 import getAllRuns, { getAllLapsPerRun } from "../api/requests";
 import { createRaceCard, hideSpinner, showSpinner } from "../dom/ui-manip";
+import "../index.scss";
 import { IKartLapsPerRun } from "../models/go-kart-types";
-import "./races.scss";
 
 let filename;
```

Comment:
Sure, just put it there for if I wanted to use the filename for something else, plus typescript wasn't complaining so i just left it😅

Reply:
If that's the case, then you can probably leave it where it is but just specify the type so that the function that is expecting a string parameter, gets a string parameter instead of a value of type 'Any'. 
It helps avoid issues in the future in case something like a number is passed down unknowingly 😊